examples/js/PlaceholderController.js

```diff
@@ -235,7 +241,7 @@ angular.module('makeithappen').controller('PlaceholderController', ['$scope', '$
                 var start = (page - 1) * this.pageSize;
                 var end = start + this.pageSize;
                 var that = this;
-                return Comments.query({_start: start, _end: end}, function (response) {
+                return toQuery.query({_start: start, _end: end}, function (response) {
```

Comment:
Yes, the `createDataProvider` function should be generic and not specific to any resource.